src/els_cth.erl

```diff
@@ -0,0 +1,37 @@
+-module(els_cth).
```

Comment:
This is quite a cryptic name, should there perhaps be a document comment saying what this module is for?

Reply:
100% agree. Do you have a suggestion?